packages/frontend/core/package.json

```diff
@@ -13,7 +13,7 @@
     "./router": "./src/router.ts",
     "./bootstrap/setup": "./src/bootstrap/setup.ts",
     "./bootstrap/register-plugins": "./src/bootstrap/register-plugins.ts",
-    "./components/pure/*": "./src/components/pure/*/index.tsx"
+    "./*": "./src/*"
```

Comment:
Use the [`imports field`](https://nodejs.org/api/packages.html#imports) to implement this feature. If you use the exports field, the reference is implemented through the symlink in node_modules. We should not rely on this behavior.